# Control law parameters from path following module
float32 y_err
geometry_msgs/Point trajectory_point
geometry_msgs/Point velocity_point
bool driving
int32 next_waypoint_idx
